packages/v4/src/content/design-guidelines/components/page/page.md

```diff
@@ -2,4 +2,58 @@
 id: Page
 section: components
 ---
-A **page** component is used to create the basic structure of a page with either vertical or horizontal navigation.
+
+The **page** component is used to define the basic layout of a page with either vertical or horizontal navigation.
+
+## Basic layouts
+PatternFly includes a flexible layout system for defining pages. Page layouts are defined using page sections as explained below. While the layout system is flexible, you should adhere to the guidelines for common elements like the masthead and page header to maintain consistency across applications.
+
+### Page sections
+Page sections allow you to define areas on a page to group content. The header, body, and footer on a page are examples of page sections. Page sections have 24px padding on left and right edges or no padding to allow the contents to extend to the edge of a page. At screen sizes smaller than 1200px, the left/right padding is reduced to 16px to create a tighter layout. Page sections can also be made sticky such that they will remain visible at the top or bottom of the viewport when the page scrolls. This is useful for creating sticky headers, footers, or other elements.
+
+The basic layout of a page differs depending on whether your application uses vertical or horizontal primary navigation as illustrated below.
+
+### Page with vertical navigation
+
+<img src="./img/page-vertical-nav.png" alt="page with vertical navigation" width="832"/>
+
+**1. Masthead:** The masthead provides a consistent header for every page in your application.
+
+**2. Navigation:** This is your primary navigation. It should be consistent across pages. See the [Navigation design guidelines](/components/navigation/design-guidelines) for more information about navigation options and usage.
+
+**3. Breadcrumb (optional):** When breadcrumbs are included, they should be the topmost element in the header section. When breadcrumbs are included on a page, the top padding is reduced to 16px. See the [Breadcrumb design guidelines](/components/breadcrumb/design-guidelines) for more information about breadcrumb usage.
```

Comment:
would add "the top padding is reduced to 16px, **from 24px**" since I don't think you mention top padding anywhere till this point!